docs/docusaurus/docs/guides/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.md

```diff
@@ -25,7 +25,7 @@ Run the following command to import the required libraries and load your DataCon
 ## Read a DataFrame and create a Checkpoint
 
 The following example uses the `read_*` method on the PandasDatasource to directly return a <TechnicalTag tag="validator" text="Validator" />. To use Validators to interactively build an Expectation Suite, see [How to create Expectations interactively in Python](/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data).
-The Validator can be passed directly to a SimpleCheckpoint
+The Validator can be passed directly to a checkpoint
```

Comment:
@donaldheppner We probably should capitalize `Checkpoint` as it is a core concept.  Thank you.

Reply:
done